src/client/components/search/Search.jsx

```diff
@@ -35,6 +35,7 @@ const Search = ({
         onChange={ handleChange }
```

Comment:
So I don't know where this bug was introduced (I'm sure not here), but I discovered it on this branch.

Before search, everything looks fine.

After search (Why'd the search move to the left?!): 
<img width="1440" alt="screen shot 2018-01-30 at 7 06 02 pm" src="https://user-images.githubusercontent.com/9269522/35603124-a2ad6ce0-05f0-11e8-9331-260ed0776680.png">

Also, things are starting to get to vertically squished.  For your todo list: add padding below the grid & maybe make the graph a little smaller

Reply:
not introduced on this branch, i noticed this in the last 1-2 days though on master i believe